config/initializers/job_configurations.rb

```diff
@@ -4,9 +4,10 @@
 cron_12m = '0/12 * * * *'
 cron_1h = '0 * * * *'
 cron_24h = '0 0 * * *'
+cron_24h_and_a_bit = '12 0 * * *' # 0000 UTC + 12 min, staggered from whatever else runs at 0000 UTC
```

Comment:
One minute is not enough of a delay, based on https://gsa-tts.slack.com/archives/CMW9H0RFX/p1724077904549099?thread_ts=1722271455.385999&cid=CMW9H0RFX 

Two bits is 25, so delaying for a bit must be ~12 minutes 🤨